project/api/serializers.py

```diff
@@ -0,0 +1,84 @@
+from rest_framework import serializers
+from api.models import Civi, Thread, Account, Category
+
+
+WRITE_ONLY = {'write_only': True}
+
+class AccountSerializer(serializers.ModelSerializer):
```

Comment:
We need to consider what fields of the user profile should be publicly available. I.e. we need to think about user privacy here. Information like first/last name, address/location, and picture are personal, so should be treated with care.